dependency.lic

```diff
@@ -700,6 +700,54 @@ class ReportbotManager
   end
 end
 
+class SlackbotManager
+  def initialize(debug)
+    @debug = debug
+    @slackbot = nil
+    @need_to_register = false
+    @messages = []
+  end
+
+  def run_queue
+    register_slackbot
+    send_messages
+  end
+
+  def register
```

Comment:
I considered making `register` take the username as an arg... that would mean we don't have to pass it around in the calling scripts, but could lead to a race condition if different scripts want to use a different username. Thoughts?